test-requirements.txt

```
@@ -4,5 +4,5 @@
 pytest
 pyyaml
 flake8
-black==20.8b1
+black==22.3.0
```

We talked through this in team and decided to not pin `black` at all
```suggestion
black
```